include/seqan3/std/span

```diff
@@ -407,8 +398,13 @@ template <class span_tp, ptrdiff_t span_extent>
     constexpr void swap(span<span_tp, span_extent> &lhs, span<span_tp, span_extent> &rhs) noexcept
     { lhs.swap(rhs); }
 
-
 //  Deduction guides
+template <ContiguousIterator iterator_t>
```

Comment:
Is this one needed? It's not (explicitly) in the proposal.

Reply:
Well, I am wondering why it is missing since it is a template argument and how would he deduce the type from the iterator otherwise? At least without it would not compile

Reply:
It makes sense that it wouldn't compile without it; just wanted to make sure.
I just compared your changes to the changes in the proposal, and this one was the only difference.